profile/README.md

```diff
@@ -1,42 +1,12 @@
 ## Welcome to the Autoware Foundation! 👋
 
-![Autoware_RViz](https://user-images.githubusercontent.com/63835446/158918717-58d6deaf-93fb-47f9-891d-e242b02cba7b.png)
+![awf_logo_white_background](https://user-images.githubusercontent.com/63835446/162110040-7cbceb06-5854-4af4-98ff-2ab77629bacc.png)
```

Comment:
```suggestion
![awf_logo_white_background](https://user-images.githubusercontent.com/63835446/162110040-7cbceb06-5854-4af4-98ff-2ab77629bacc.png | width=200)
```

I'm not able to check how much the width should exactly be but current one is too big. Added 200 for testing.

Reply:
@xmfcx Oh, I didn't know this feature! Just out of curiosity, is this GitHub's extension? Or supported in the original Markdown spec? (If you know that.)

Reply:
@kenji-miyake I've looked it up from https://gist.github.com/uupaa/f77d2bcf4dc7a294d109

Reply:
I tried that and it didn't seem to work when previewing the file. What did work was to upload the file, copy the file URL and then replace the markdown code with a HTML `img` tag that contains a width attribute.

> I'm not able to check how much the width should exactly be but current one is too big. Added 200 for testing.

So, I tried 200 pixels and it was very small. For comparison, I checked the GitHub org profile page and their banner was as wide as the page, so that seems standard. 

@xmfcx If you could explain what you mean by `too big` and what would be more acceptable, then hopefully we can find something that works better.

Reply:
For comparison, GitHub's banner image is 2000x800 and the AWF image that I added is 1287x597, and I believe that the website handles resizing of the image based on the device being used. 

(we probably need to look into creating an updated banner image for the AWF that is a sensible size, but that's a different problem)